src/fetch_business_partners_info.py

```diff
@@ -0,0 +1,104 @@
+import os
+import json
+import datetime
+import time
+import pandas as pd
+from unicodedata import normalize
+from selenium import webdriver
+from bs4 import BeautifulSoup
+
+
+def selenium_driver_path():
+    for path in ['chromedriver.exe', 'chromedriver']:
+        if os.path.exists(path):
+            return path
+    raise BaseException('ChromeDriver not found')
+
+
+def normalize_name(name):
+    return normalize('NFKD', name).encode('ASCII', 'ignore').decode('utf-8').lower()
+
+
+def make_google_cache_url(name):
+    return 'http://webcache.googleusercontent.com/search?btnG=Search&q=cache:www.consultasocio.com/q/sa/%s' % \
+           normalize_name(name).replace(' ', '-')
+
+
+def make_consulta_socio_url(name):
+    return 'http://www.consultasocio.com/q/sa/%s' % normalize_name(name).replace(' ', '-')
+
+
+def extract_info_from_page(url, info, driver, error_msg):
+    # Open the page using Selenium in order to collect the full HTML to be parsed by BeautifulSoup
+    driver.get(url)
+    driver.switch_to.default_content()
+    soup = BeautifulSoup(driver.page_source, 'html.parser')
+
+    # Somethimes google can ask for captcha, so you should type it!
+    if 'To continue, please type the characters below:' in soup.text:
+        while 'To continue, please type the characters below:' in soup.text:
+            driver.switch_to.default_content()
+            soup = BeautifulSoup(driver.page_source, 'html.parser')
+            time.sleep(1)
+
+    # Collect the information if there is no error
+    if error_msg.lower() not in soup.title.text.lower():
+        for partner in soup.findAll('span', attrs={'class': 'socio'}):
+            info['partners'].append(partner.text)
+        for company_info in soup.findAll('p', attrs={'class': 'number'}):
+            info['business'].append(company_info.find('span', {'class': 'text'}).text)
+        try:
+            info['cache_info'] = soup.find('div', attrs={'id': 'google-cache-hdr'}).text
+        except:
+            info['cache_info'] = str(datetime.datetime.now())
+        print(info)
+        return info
+
+
+def fetch_business_infos():
+    file_base_name = '{}-congressperson-business-partners.json'.format(datetime.date.today().strftime('%Y-%m-%d'))
+    output_file_path = os.path.join(os.path.abspath('../data'), file_base_name)
```

Comment:
I got your point but in Python it is a common practice to run scripts form the root of the project. Beyond that all other files from `src/` behave like that. It's a kind of standard for this (and most of Python) projects.

If you can implement a way that builds the path from the real location of the `.py` it's fine, it would work with `python fetch_business_partners_info.py` and `python data/fetch_business_partners_info.py`— otherwise can you make your script coherent with all its future buddies from `src/`?